guides/common/modules/con_understanding-package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,32 @@
+[id="understanding_package_mode_and_image_mode_hosts_{context}"]
+= Understanding Package mode and Image mode hosts
+
+You can operate hosts in two modes: package mode and image mode.
+
+.Image mode hosts
+Image mode hosts include additional fields that reflect their boot status.
+These fields track the container images used in various scenarios:
+
+* *Booted Image & Digest*: Indicates the container image the host is currently using.
+* *Staged Image & Digest*: Indicates the container image the host uses after the next reboot.
+* *Available Image & Digest*: Represents the container image cached by running `bootc upgrade --check`.
+* *Rollback Image & Digest*: Represents the container image the host reverts to after the next reboot if `Bootc rollback` is applied.
+
+.All host listing page updates
+* *New Column for Host Types*: On the new Hosts Overview page, the host listing page includes a column with icons that indicate the host type.
+Note that users can add or remove the image mode column by using the *Manage columns* feature.
+** Clicking an icon displays the image path and digest.
+** Note that the image path can be used directly with podman or docker to pull the image from a registry.
+* *Search Enhancements*:
+** Use `image_mode=true` or `image_mode=false` to filter hosts by type.
+** Unregistered systems are assumed to be in package mode.
+** Search using eight new fields:
+`Bootc_booted_image`, `Bootc_booted_digest`, `Bootc_staged_image`, `Bootc_staged_digest`, `Bootc_available_image`, `Bootc_available_digest`, `Bootc_rollback_image`, `Bootc_rollback_digest`.
+
+.Host Details Page Enhancements
+A new card appears on the Details tab for registered hosts, displaying the additional fields from `Bootc`.
+
+.Important Notes
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, related packages must be included in the image mode host's container image.
```

Comment:
Perhaps this should be a prerequisite in the procedure? (I'm not sure because it isn't clear to me why that's needed.)